docs/dive-in/2024/02/2024-02-02-neon-db.md

```diff
@@ -0,0 +1,69 @@
+# Store data
+
+- Author: Magali Deslous-Paoli
+- Date : 2024-02-02
+
+## Summary
+
+Store our data in a shared DB
+
+## Context
+
+We will need to store our data into a shared database, accessible from our deployment. We do not need a lot of storage, but we do need simplicity and availability
+
+## Solution
+
+### NeonDB
+
+Neon is an open source serverless Postgres with a free plan available. It includes some useful features such as the possibility of creating branches of our dabatase.
+
+Neon free tier plan allows:
+
+- 10 branches
+- 3GiB per branch
+- 100h hour of Active time per month
```

Comment:
What does this mean, if we exceed 100h does our application crash?

Reply:
If we exceed our time, only our primary branch (prod) would be available, but not our others branches. So no, our application would not crash, but we could not use the db for testing purposes